src/RestControllers/RestControllerHelper.php

```diff
@@ -38,7 +37,13 @@ public static function validationHandler($validationResult)
     {
         if (property_exists($validationResult, 'isValid') && !$validationResult->isValid()) {
             http_response_code(400);
-            return $validationResult->getMessages();
+
+            if (property_exists($validationResult, 'getValidationMessages')) {
```

Comment:
This may not be an update which "sticks" in the final FHIR implementation, but it allows the code to support "un-updated" endpoints and the patient endpoint.